sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1976,6 +1976,17 @@ val SHUFFLE_COMPRESSION_LZ4_CHUNK_SIZE = conf("spark.rapids.shuffle.compression.
       .integerConf
       .createWithDefault(20)
 
+  val SHUFFLE_PARTITIONING_MAX_CPU_BATCH_SIZE =
```

Comment:
nit: make this an internal config? `.internal()`

Reply:
fixed